modules/layers/src/types.ts

```diff
@@ -0,0 +1,6 @@
+export type MaterialProps = {
+  ambient: number;
+  diffuse: number;
+  shininess: number;
+  specularColor: [r: number, g: number, b: number];
```

Comment:
`@babel/preset-typescript` does not support named tuples and this breaks local debugging with the examples.